### DISCUSSION TITLE
Avast and some other antivir programs report dude.exe as infected with trojan

### DISCUSSION THREAD

**Discussion by Eltombek** [PJDude/dude#9] [Q&A] [answered]
Avast and some other antivir programs report dude.exe as infected with trojan. Additionally windows defender prevented the file execution as there is no issuer.

**Comment by PJDude** [comment, marked as answer]
This is typical issue for distributions made with Pyinstaller or Nuitka: [pyinstaller-false-positive](https://discuss.python.org/t/pyinstaller-false-positive/43171). Dude releases are made in GitHub containers, I don't expect any malware there ...
There is always possibility to check any file with [virustotal](https://www.virustotal.com/gui/home/upload) or maybe similar services. As You can see many respected antivirus don't report problems with dude binaries. I don't have certificate to digitally sign windows binaries also submitting binaries for every anti-virus company per every release of such niche project is too much hassle.
